dear_petition/petition/migrations/0019_auto_20200407_1720.py

```diff
@@ -0,0 +1,55 @@
+# Generated by Django 2.2.10 on 2020-04-07 16:49
+
+from django.db import migrations
+
+
+def update_existing_ciprs_records(apps, schema_editor):
+    CIPRSRecord = apps.get_model("petition", "CIPRSRecord")
+    records = CIPRSRecord.objects.all()
+    for r in records:
+        refresh_record_from_data(r)
+
+
+def refresh_record_from_data(record):
+    """Updates model fields that depends on the data JSONField
+
+    The record exists, but the raw data (data - JSONFIELD) has
+    changed. Let's update the models that are extracting data
+    from this field.
+    """
+    record.file_no = self.data["General"].get("File No", "")
+    record.county = record.data["General"].get("County", "")
+    record.dob = record.data["Defendant"].get("Date of Birth/Estimated Age", "")
+    record.sex = record.data["Defendant"].get("Sex", "")
+    record.race = record.data["Defendant"].get("Race", "")
+    record.case_status = record.data["Case Information"].get("Case Status", "")
+    record.offense_date = make_date_obj_aware(
+        record.data["Case Information"].get("Offense Date", "")
+    )
+    record.arrest_date = record.data["Offense Record"].get(
+        "Arrest Date", dt_obj_to_date(record.offense_date)
+    )
+    record.jurisdiction = get_jurisdiction(record)
+    record.save()
+
+
+def get_jurisdiction(record):
+    is_superior = record.data["General"].get("Superior", "")
+    is_district = record.data["General"].get("District", "")
+    if is_superior:
+        return SUPERIOR_COURT
```

Comment:
It looks like `SUPERIOR_COURT`, `DISTRICT_COURT`, and `NOT_AVAILABLE` are not defined in this file.

Reply:
thanks done.